src/apps/common/submodules/numbers/numbers.js

```diff
@@ -69,13 +64,13 @@ define(function(require) {
 
 				var arrayNumbers = data.listAccounts.length ? data.listAccounts[0].usedNumbers : [];
```

Comment:
I forgot this one, the `arrayNumbers` declaration can be moved to the previous variable declaration block (https://docs.2600hz.com/ui/docs/javascriptStyleGuide/#variables).